src/Components/Comments.jsx

```diff
@@ -1,7 +1,29 @@
-export const Comments = () => {
+import { useState } from "react";
+import { useEffect } from "react";
+import { getReviewComments } from "../Utils/api";
+
+export const Comments = ({ review_id }) => {
+  const [comments, setComments] = useState([]);
+  useEffect(() => {
+    getReviewComments(review_id).then(({ comments }) => {
+      setComments(comments);
+    });
+  }, []);
+
   return (
-    <section>
-      <p>Comments</p>
-    </section>
+    <ul>
```

Comment:
1. let's add a loading message for whilst we are waiting for comments.

2. what if there are no comments (yet) for the review?  Let's have a message letting our user know that